Examples/Tests/pass_mpi_communicator/PICMI_inputs_2d.py

```diff
@@ -0,0 +1,254 @@
+# --- Test if MPI_communicator is correctly passed from Python to CPP
+# --- This test should be run with MPI enabled
+# --- Inputs taken from langmuir_2d. Runs 1 step, and will check
+# --- if the correct amount of processors are initialized in AMReX.
+
+from mpi4py import MPI
+from pywarpx import picmi
+constants = picmi.constants
+# from pywarpx import geometry
+# # need to set geometry before importing _libwarpx
+# geometry.coord_sys = 0
+# geometry.prob_lo = [0]*2
+
+
+##########################
+# MPI communicator setup
+##########################
+
+# must initialize everything separately on separate processors
+# so we need to create comm first before setting params,
+# and importing packages
+
+# split processor 0 into separate communicator from others
+comm_world = MPI.COMM_WORLD
+rank = comm_world.Get_rank()
+if rank == 0:
+    color = 0
+else:
+    color = 1
+new_comm = comm_world.Split(color)
+
+if color == 0:
```

Comment:
Instead of duplicating all of the setup, could you do something like
`plasma_density = [1.e18, 1.e19][color]`
and
`diagname = f'diag{color+1}'`
and have both processors run the same code?

Reply:
I'll fix that. Yeah I've tested with both processors running the same code, and they get the same outputs as expected.